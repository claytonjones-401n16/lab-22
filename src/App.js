import React from 'react';

import Header from './components/Header.js';
import Footer from './components/Footer.js';
import Counter from './components/Counter.js';

import './styles.scss';

class App extends React.Component {
    render() {
        return (
            <React.Fragment>
                <Header />
                <main>
                    <Counter />
                </main>
                <Footer />
            </React.Fragment>
        );
    }
}

export default App;
