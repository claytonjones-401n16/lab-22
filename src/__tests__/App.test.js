import React from 'react';

import renderer from 'react-test-renderer';

import App from '../App';

describe('The app looks as expected', () => {
  it('matches the snapshot', () => {
    let html = renderer.create(<App />).toJSON();
    expect(html).toMatchSnapshot();
  });
});