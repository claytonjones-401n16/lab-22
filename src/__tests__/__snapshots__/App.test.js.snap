// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The app looks as expected matches the snapshot 1`] = `
Array [
  <header>
    <h1>
      Counters R Fun
    </h1>
  </header>,
  <main>
    <section
      className="counter"
    >
      <span
        className="down clicker"
        onClick={[Function]}
      >
         
        -
         
      </span>
      <span
        className="count "
      >
        0
      </span>
      <span
        className="up clicker"
        onClick={[Function]}
      >
         
        +
         
      </span>
    </section>
  </main>,
  <footer>
    Â© 2018 Code Fellows
  </footer>,
]
`;
